src/components/DateBox/index.js

```diff
@@ -1,8 +1,36 @@
 import { Flex } from "@chakra-ui/react";
 import { ChevronLeftIcon, ChevronRightIcon } from "@chakra-ui/icons";
 import { Box, Container, Spacer, Text } from "@chakra-ui/react";
+import { useState } from "react";
+import {
+  add,
+  format,
+  isBefore,
+  isToday,
+  isTomorrow,
+  isYesterday,
+  sub,
+} from "date-fns";
+
+const DateBox = () => {
+  const [date, setDate] = useState(new Date());
+
+  console.log(date);
+
+  const formatDate = (date) => {
+    if (isToday(date)) {
+      return "Today";
+    }
+    if (isYesterday(date)) {
+      return "Yesterday";
+    }
+    if (isTomorrow(date)) {
+      return "Tomorrow";
+    }
+
+    return format(date, "MM/dd/yyyy");
```

Comment:
"MM/dd/yyyy" is not the format mentioned in the Jira ticket, can you update?

Reply:
donnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnne